config/spryk/spryks/Spryker/Glue/AddGlueDeleteResource.yml

```diff
@@ -0,0 +1,78 @@
+spryk: structure
```

Comment:
https://github.com/spryker-sdk/spryk/pull/21#discussion_r311569539

same steps gave me:

```
Enter value for AddGlueDependencyClientBridge.dependentModuleOrganization argument [Spryker]:
```
I would assume this would be called ClientModuleOrganization? or maybe clientModule should have a hint like Organization.ModuleName